fmriprep/viz/reports.py

```diff
@@ -120,24 +118,59 @@ def index(self):
         for sub_report in self.sub_reports:
             sub_report.order_by_run()
 
+        subject_dir = self.root.split('/')[-1]
+        subject = re.search('^(?P<subject_id>sub-[a-zA-Z0-9]+)$', subject_dir).group()
+        error_dir = os.path.join(self.root, '../../log', subject[4:])
+        if os.path.isdir(error_dir):
+            self.index_error_dir(error_dir)
+
+    def index_error_dir(self, error_dir):
+        for root, directories, filenames in os.walk(error_dir):
+            for f in filenames:
+                try:
+                    crash_data = loadcrash(os.path.join(root, f))
+                except ValueError:
+                    continue
```

Comment:
I would prefer to avoid silencing all ValueError exceptions and limit the files that are attempted to be read to those that begin with "crash" and end with ".pklz"